build.gradle.kts

```diff
@@ -0,0 +1,156 @@
+/*
+ * This file was generated by the Gradle 'init' task.
+ */
+
+plugins {
+  `java-platform`
+  `maven-publish`
+}
+
+defaultTasks("publishKnotxPublicationToMavenLocal")
+
+tasks {
+  getByName("build").dependsOn("publishKnotxPublicationToMavenLocal")
+}
+
+repositories {
+  jcenter()
+  mavenLocal()
+  maven(url = "https://oss.sonatype.org/content/repositories/snapshots")
+  maven(url = "http://repo.maven.apache.org/maven2")
+}
+
+group = "io.knotx"
+version = "2.1.1-SNAPSHOT"
```

Comment:
Can we move `version` to `gradle.properties` as we do usually in other projects?